### ISSUE TITLE
Fix false negatives on file's issues when the first line is empty

### ISSUE THREAD

**Issue by abbas-sabra-sonarsource** [SonarSource/sonarlint-visualstudio#2860]
### Description

Issues reported with `endline==0 `is meant to be issues on the entire file. In SLVS these issues are reported on the entire first line. That is okay if the first line is not empty. But if the first line is empty this issue will not be shown and the rule will not trigger.

### Reproduction steps
1- enable a file level rule like `cpp:CommentMixedStyles`:
>    "cpp:CommentMixedStyles": {
>       "level": "On"
>     },

2-  analyzed this file. CommentMixedStyles is raised on the first line:

```
// the rule is raised on this line
typedef int x1;

/*      */
//
```

3- Make the first line empty, and the rule will no longer be raised:

```

// the issue is not raised on this file
typedef int x1;

/*      */
//
```

### Expected results
The issue appears in the Error List and there are squigglies under the opening comment in the editor.

### Actual results
The issue does not appear in the Error List. There are no  squigglies in the editor.

### Affected versions:
?All SLVS versions to date that support C++ analysis.
Affected languages: C/C++

**Comment by duncanp-sonar** [comment]
@abbas-sabra-sonarsource, @ugras-ergun-sonarsource I've created a more general new issue to replace this one. See #2963.
I'll close this one.
